webui/react/src/types.ts

```diff
@@ -241,6 +241,7 @@ export interface Agent {
   registeredTime: number;
   resourcePools: string[];
   slots?: SlotsRecord;
+  slotStats?: Api.V1SlotStats;
```

Comment:
should always be available. but might be empty

Reply:
@hamidzr https://github.com/determined-ai/determined/pull/9048/files#diff-674d4524dc8c6e0f5971b4fcd5d6b220a2a5e05bbcf55954f9faf7883136cc1bR83, `slot_stats` should be added in `required` in protobuf because `slotStats` can be `undefined` now [code](https://github.com/determined-ai/determined/pull/9048/files#diff-58691f26d68f9a72c8a29a86d16b6cdb90e2719ce1e34b745bab5b6073b044fcR1098). will comment the same thing in your PR just in case

Reply:
fixed